Cookbook/Technical-Documents/XcodeTips.md

```diff
@@ -0,0 +1,33 @@
+## Tips and tricks
+
+### Errors when pulling code/switching branches
+Various errors can occur when pulling code/switching branches. 
+Typically all that is needed is running `bundle exec pod install` and closing
+and reopening the project in Xcode. The following directions are
+intended to be a more thorough reset to deal with bigger changes.
+
+- Clean Build: Product > Clean (shortcut: cmd + shift + k)
+- Close Xcode
+- Update bundle: run `bundle install`
+- Update pods: `bundle exec pod deintegrate && bundle exec pod install --repo-update`
```

Comment:
From the comments in a recent Slack thread (https://babylonhealth.slack.com/archives/CFLN31FM5/p1556724542020400), deintegrate is not recommended, except as a last resort, as it can have some negative side-effects.

Reply:
Yes. Trying to figure out what the preferred solution actually is before updating this PR.

Reply:
Done